templates/support/community-support.html

```diff
@@ -29,7 +29,7 @@ <h1>Community support</h1>
 <div class="p-strip is-deep">
   <div class="row u-equal-height">
     <div class="col-4 p-card">
-      <h3><a href="https://discourse.ubuntu.com/t/community-support/709">Find community support</a></h3>
+      <h3><a href="https://ubuntu.com/community/support">Find community support</a></h3>
```

Comment:
```suggestion
      <h3><a href="/community/support">Find community support</a></h3>
```

When it's an internal link you need only set the path as the href

Reply:
Got it! Thank you for letting me know.